rewrite-java/src/main/java/org/openrewrite/java/RenameVariable.java

```diff
@@ -38,35 +45,95 @@ public RenameVariable(J.VariableDeclarations.NamedVariable variable, String toNa
 
     private class RenameVariableByCursor extends JavaIsoVisitor<P> {
         private final Cursor scope;
-        private boolean isConstant = false;
+        private final Stack<Tree> currentNameScope;
+        private @Nullable Tree currentBlock;
```

Comment:
For sake of consistency, would appreciate @Nullable being in front of modifier. When @Nullable, leave a blank line between this field and the one above/below it.

Reply:
Okay, sounds good - moved @Nullable to the front of the modified and added a new line.